Dockerfile.bot

```diff
@@ -0,0 +1,25 @@
+# Use this Dockerfile to setup the development environment for bot
+FROM golang:1.10.2-alpine3.7 AS dev
```

Comment:
gonna make Dockerfile.prod once i get redis working, so this just dev env

Reply:
why do you need redis as lead developer i deserve to understand why you want to store values

Reply:
crappy lead, add redis or else

Reply:
no